aws/templates/application/application_lambda.ftl

```diff
@@ -73,7 +72,7 @@
                                     source=linkTargetRoles.Inbound["invoke"]
                                 /]
                             [/#if]
-                            [#break]
+                        [#break]
```

Comment:
This looks like the wrong level of indenting

Reply:
Fixed